modules/virt-about-upgrading-virt.adoc

```diff
@@ -24,10 +24,7 @@ connection. Most automatic updates complete within fifteen minutes.
 [id="how-upgrades-affect-cluster_{context}"]
 == How {VirtProductName} upgrades affect your cluster
 
-* Upgrading does not interrupt virtual machine workloads.
-** Virtual machine pods are not restarted or migrated during an upgrade. If you
-need to update the `virt-launcher` pod, you must restart or live migrate the
-virtual machine.
+* When you upgrade {VirtProductName}, virtual machine workloads including `libvirt`, `virt-launcher`, and `qemu` are updated automatically.
```

Comment:
This is not always true: now we removed the `evict` strategy form the default configuration so VMs that cannot be live-migrated are still not going to be automatically updated (at least not with the default configuration).

Reply:
Rephrasing to `* When you upgrade {VirtProductName}, virtual machine workloads including `libvirt`, `virt-launcher`, and `qemu` update automatically if they support live migration.`